src/test/components/TrackScreenshots.test.js

```diff
@@ -325,6 +340,17 @@ function setup(
     return parseInt(screenshotHover().style.top);
   }
 
+  function moveMouseAndGetSize(
```

Comment:
nit: not so important, but maybe it would be better to also mention the size of what in the function name like `moveMouseAndGetScreenshotSize`. What do you think?